types/chrome-har-capturer/index.d.ts

```diff
@@ -0,0 +1,168 @@
+/// <reference types="node" />
+
+import { EventEmitter } from "events";
+
+export interface Page<U> {
+    id: string;
+    title: any;
```

Comment:
It does seem like all of these `anys` could have proper types, but it isn't a complete requirement.